schema/src/main/jade-tables/hles_dog.table.json

```diff
@@ -243,22 +334,175 @@
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_4_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_4_weeks",
+      "datatype": "integer"
+    },
+    {
+      "name": "dd_alternate_recent_residence_5_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_5_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_5_weeks",
+      "datatype": "integer"
+    },
+    {
+      "name": "dd_alternate_recent_residence_6_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_6_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_6_weeks",
+      "datatype": "integer"
+    },
     {
-      "name": "dd_other_recent_short_term_residences",
-      "struct_name": "hles_dog_residence",
-      "type": "repeated"
+      "name": "dd_alternate_recent_residence_7_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_7_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_7_weeks",
+      "datatype": "integer"
+    },
+    {
+      "name": "dd_alternate_recent_residence_8_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_8_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_8_weeks",
+      "datatype": "integer"
+    },
+    {
+      "name": "dd_alternate_recent_residence_9_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_9_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_9_weeks",
+      "datatype": "integer"
+    },
+    {
+      "name": "dd_alternate_recent_residence_10_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_10_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_10_weeks",
+      "datatype": "integer"
+    },
+    {
+      "name": "dd_alternate_recent_residence_11_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_11_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_11_weeks",
+      "datatype": "integer"
+    },
+    {
+      "name": "dd_alternate_recent_residence_12_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_12_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_12_weeks",
+      "datatype": "integer"
+    },
+    {
+      "name": "dd_alternate_recent_residence_13_state",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_13_zip",
+      "datatype": "string"
+    },
+    {
+      "name": "dd_alternate_recent_residence_13_weeks",
```

Comment:
Do we know we won't see more than 13 alternate addresses? Or is that just the highest you've seen so in the data so far?

Reply:
The RedCap survey itself encodes 13 as the upper bound (which is why we have to read differently-named raw fields for each). If it was truly unbounded, we'd have no choice but to use an array here.